tests/core/common/test_endpoint_broadcast.py

```diff
@@ -0,0 +1,40 @@
+from lahja import BaseEvent, ConnectionConfig
+from lahja.tools import drivers as d
```

Comment:
meh single letter alias

Reply:
Actually, looking at the saved space we gain, it's not that bade to make an exception here!